### ISSUE TITLE
Auto-generate short methods section?

### ISSUE THREAD

**Issue by kcranston** [HullUni-bioinformatics/ReproPhylo#59]
It would be great if ReproPhylo could auto-generate a short methods section for an analysis. This would 1) save time for the author and 2) include data about versions, options, etc that authors might not otherwise include. Not sure how feasible this is - whether you could template such text or if analyses are just too diverse.

**Comment by szitenberg** [comment]
Yes, this is definitely a goal. As of now, as a first easy step reprophylo produces a separate text line and reference for each process in the html report, e.g., 

<pre>
The alignment(s) x, y and z were trimmed using the program trimAl 1.2rev59 [1].

Reference:
Salvador Capella-Gutierrez; Jose M. Silla-Martinez; Toni Gabaldon. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics 2009 25: 1972-1973.
</pre> 

but we plan to bunch these up in a nice way to form a coherent paragraph with more details about the parameter choices. Currently, the parameters can only be seen in the explicit command lines, also printed in the html report.
